invader_invoice_payment/models/account_invoice.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    transaction_ids = fields.Many2many(
+        comodel_name="payment.transaction",
+        relation="account_invoice_transaction_rel",
+        column1="account_invoice_id",
```

Comment:
@acsonefho Could you just use the same name as in further versions of Odoo (invoice_id) ?

https://github.com/odoo/odoo/blob/12.0/addons/payment/models/payment_acquirer.py#L625